Makefile

```diff
@@ -39,11 +39,11 @@ mkimage:
 		.
 
 .PHONY: iso
-iso: nerdctl-$(NERDCTL_VERSION)-$(ARCH) qemu-$(QEMU_VERSION)-copying cri-dockerd-$(CRI_DOCKERD_VERSION)-$(ARCH) layer
```

Comment:
`layer` is actually being used to generate the ubuntu container OCI archive, it should not be deleted.